src/main/webapp/app/lecture/overview/course-lectures/video-unit/video-unit.component.ts

```
@@ -1,54 +1,69 @@
-    imports: [LectureUnitComponent, SafeResourceUrlPipe],
+    standalone: true,
+    imports: [CommonModule, VideoPlayerComponent, LectureUnitComponent, SafeResourceUrlPipe],
     templateUrl: './video-unit.component.html',
 })
 export class VideoUnitComponent extends LectureUnitDirective<VideoUnit> {
     protected readonly faVideo = faVideo;
 
-    // List of regexes that should not be blocked by js-video-url-parser
     private readonly videoUrlAllowList = [
-        // TUM-Live. Example: 'https://live.rbg.tum.de/w/test/26?video_only=1'
         RegExp('^https://live\\.rbg\\.tum\\.de/w/\\w+/\\d+(/(CAM|COMB|PRES))?\\?video_only=1$'),
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)